modules/op-configuring-pipelines-as-code-for-a-github-app.adoc

```diff
@@ -0,0 +1,79 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="configuring-pipelines-as-code-for-a-github-app_{context}"]
+= Configuring Pipelines as Code for a GitHub App 
+
+[role="_abstract"]
+GitHub Apps act as a point of integration with {pipelines-title}, and brings the advantage of Git based workflows to Tekton Pipelines. Cluster administrators can configure a single GitHub application for all users of the cluster. For GitHub Apps to work with Pipelines as Code, ensure that the webhook of the GitHub App to points to your Pipelines as Code controller route (or ingress endpoint) which listens for GitHub events.
+
+[id="configuring-github-app-for-pac"]
+== Configuring a GitHub App
+
+[discrete]
+.Procedure
+
+. Sign in to your GitHub account, go to **Settings** --> **Developer settings** --> **GitHub Apps**, and click **New GitHub App**.
+
+. Provide the following information in the GitHub App form: 
+
```

Comment:
Shouldn't the nested bulleted lists in this procedure use a single `*` rather than `**`. Is there a reason to use `**` rather than `*`?